package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "carousel_module",
+  "version": "1.0.0",
+  "description": "Component to handle the main photo gallery of the checkout page.",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack -d -w",
+    "build": "nodemon server/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/abode-ly/carousel_module.git"
+  },
+  "keywords": [],
+  "author": "Toshihiro Akabori",
+  "license": "",
+  "bugs": {
+    "url": "https://github.com/abode-ly/carousel_module/issues"
+  },
+  "homepage": "https://github.com/abode-ly/carousel_module#readme",
+  "dependencies": {
+    "bootstrap": "^4.3.1",
```

Comment:
Let's not use bootstrap! We should use CSS that we write. Bootstrap is great for prototyping but we want to learn how to do our own CSS

Reply:
@chinkevin25 
Yes, we've heard about this and have removed it from dependencies in the following PR.
Thanks.